int32 request
---
geometry_msgs/PoseArray poses
std_msgs/Int32MultiArray region1
std_msgs/Int32MultiArray region2